cmd/server/main.go

```diff
@@ -31,21 +31,38 @@ func Init() string {
 func main() {
 	address := Init()
 	r := router.New()
+
+	// The index endpoint displays information about the bark server.
 	r.GET("/", controllers.IndexController)
+
+	// This is a demo endpoint to ensure bark server is running which will print Hello, `name`!.
 	r.GET("/hello/{name}", Hello)
+
+	// Bark client contains the logic which decides which out of the two (single/multiple) insertion endpoints is called.
+	// This endpoint sends single log entry at a time to the DB.
 	r.POST("/insertSingle", controllers.SendSingleToChannel)
+	// This endpoint handles the batch insertion of logs to the DB.
 	r.POST("/insertMultiple", controllers.SendMultipleToChannel)
+
+	// This endpoint is responsible to initiate server shut down, following which Bark server will not process any new incoming requests.
+	// It will, however, shut down after it has completely sent all the logs received up till that point of time.
```

Comment:
We should use the words "has completely saved all logs received till that point to the database" - makes it more clear. The word "sent" makes me feel like the logs are being further forwarded to some other place.

Reply:
Worded the comment in a more comprehensible way as suggested.